README.md

```diff
@@ -82,6 +82,8 @@ cd chapter
 Install dependencies:
 ```
 npm install
+cd client
```

Comment:
Not sure it matters, but tomn on Discord used `cd ./client`

As far as I know they are the equivalent, but if there's an OS-specific reason to include the period, then we could consider it.

Reply:
Agree. Made the suggested change.